### ISSUE TITLE
@wdio/mocha-framework 7.7.4 - DeprecationWarning

### ISSUE THREAD

**Issue by unickq** [webdriverio/webdriverio#7040]
After upgrading @wdio/mocha-framework from 7.7.3 to 7.7.4 got this DeprecationWarning.

`npx wdio --spec integration-tests\integration-tests\t1.spec.ts `
```
[0-1] RUNNING in chrome - integration-tests\integration-tests\t1.spec.ts
[0-1] (node:18672) DeprecationWarning: "Runner(suite: Suite, delay: boolean)" is deprecated. Use "Runner(suite: Suite, {delay: boolean})" instead.
[0-1] (Use `node --trace-deprecation ...` to show where the warning was created)
```

Probably related to https://github.com/webdriverio/webdriverio/pull/6967

```json
"dependencies": {
    "mocha": "^9.0.1",
    "@wdio/cli": "^7.7.4",
    "@wdio/junit-reporter": "^7.7.3",
    "@wdio/local-runner": "^7.7.4",
    "@wdio/mocha-framework": "^7.7.4",
    "@wdio/selenium-standalone-service": "^7.7.4",
    "@wdio/spec-reporter": "^7.7.3",
    "@wdio/sync": "^7.7.4",
}
```

**Comment by christian-bromann** [comment]
@unickq you want to help us take a stab at this?

**Comment by unickq** [comment]
@christian-bromann yes, added https://github.com/webdriverio/webdriverio/issues/7040

**Comment by christian-bromann** [comment]
Let's continue convo in #7047

**Comment by unickq** [comment]
Reopen by https://github.com/webdriverio/webdriverio/pull/7047#issuecomment-866696954

tldr:
https://www.npmjs.com/package/@types/mocha v9 support required

**Comment by tehhowch** [comment]
Upstream PR for type support is here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54208

**Comment by christian-bromann** [comment]
Closing, thanks @unickq
